pallets/subtensor/src/tests/epoch.rs

```diff
@@ -2736,106 +2736,8 @@ fn test_blocks_since_last_step() {
         assert_eq!(SubtensorModule::get_blocks_since_last_step(netuid), 27);
     });
 }
-// // Map the retention graph for consensus guarantees with an single epoch on a graph with 512 nodes, of which the first 64 are validators, the graph is split into a major and minor set, each setting specific weight on itself and the complement on the other.
```

Comment:
Do not delete this. It generates code graph for yuma consensus guarantees

Reply:
You mean the commented code?

Reply:
yah. its what we used to generate the graphs in `docs/consensus.md`

Reply:
it's better to have these things somewhere outside or better - encapsulate it into a script. eventually commented code will be inevitably deleted. not by me, then by someone else.